articles/active-directory/fundamentals/what-is-deprecated.md

```diff
@@ -15,7 +15,7 @@ ms.custom: it-pro, has-azure-ad-ps-ref
 
 # What's deprecated in Microsoft Entra ID?
 
-The lifecycle of functionality, features, and services are governed by policy, support timelines, data, also leadership and engineering team decisions. Lifecycle information allows customers to predictably plan long-term deployment aspects, transition from outdated to new technology, and help improve business outcomes. Use the definitions below to understand the following table with change information about Azure Active Directory (Azure AD) and Microsoft Entra features, services, and functionality. 
+The lifecycle of functionality, features, and services are governed by policy, support timelines, data, also leadership and engineering team decisions. Lifecycle information allows customers to predictably plan long-term deployment aspects, transition from outdated to new technology, and help improve business outcomes. Use the definitions below to understand the following table with change information about Microsoft Entra ID and Microsoft Entra features, services, and functionality. 
```

Comment:
```suggestion
The lifecycle of functionality, features, and services are governed by policy, support timelines, data, also leadership and engineering team decisions. Lifecycle information allows customers to predictably plan long-term deployment aspects, transition from outdated to new technology, and help improve business outcomes. Use the definitions below to understand the following table with change information about Azure Active Directory (Azure AD) and Microsoft Entra features, services, and functionality. 
```

We need to keep the Azure AD reference here, as we're talking about deprecated Azure AD features before the rebrand.